lib/addons/flagger/index.ts

```diff
@@ -0,0 +1,73 @@
+import * as blueprints from '../../';
+import { Construct } from 'constructs';
+import { Values } from "../../spi";
+import merge from "ts-deepmerge";
+
+/**
+ * User provided options for the FlaggerAddonProps values.
+ */
+export interface FlaggerAddOnProps extends blueprints.HelmAddOnUserProps {
+  prometheusInstall?: boolean;
```

Comment:
let's rename to `installPrometheus` to keep it consistent with all flags like that <verb><subject>, e.g.

https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/addons/appmesh/index.ts#L17

https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/addons/cluster-autoscaler/index.ts#L24